src/components/Dialog/Dialog.scss

```diff
@@ -73,43 +117,72 @@
   }
 }
 
-.tm-w-m {
-  .tm-dialog {
-    left: 350px;
+.tm-h-m {
```

Comment:
Move this class def just after .tm-h-xs

Reply:
Lint won't accept that, for descending specificity

Reply:
if lint said that